apps/system-e2e/src/tests/judicial-system/smoke/search-warrant-tests.spec.ts

```diff
@@ -90,4 +91,59 @@ test.describe.serial('Search warrant tests', () => {
     await page.getByRole('button', { name: 'Loka glugga' }).click()
     await expect(page).toHaveURL('/krofur')
   })
+
+  test.describe.serial('Search warrant court tests', () => {
+    test('court should receive search warrant request and make a ruling', async ({
+      judgePage,
+    }) => {
+      const page = judgePage
+      const yesterday = subDays(new Date(), 1).toLocaleDateString('is-IS', {
+        month: '2-digit',
+        day: '2-digit',
+        year: 'numeric',
+      })
+
+      await page.goto(`domur/rannsoknarheimild/mottaka/${caseId}`)
+      await page.getByTestId('courtCaseNumber').click()
+      await page.getByTestId('courtCaseNumber').fill(randomCourtCaseNumber())
+      await page
+        .getByText('Veldu dómara/aðstoðarmann *Veldu héraðsdómara')
+        .click()
+      await page.locator('#react-select-judge-option-1').click()
+      await page.getByTestId('continueButton').click()
+      await expect(page).toHaveURL(`domur/rannsoknarheimild/yfirlit/${caseId}`)
```

Comment:
Did you forget the root slash?
```suggestion
      await expect(page).toHaveURL(`/domur/rannsoknarheimild/yfirlit/${caseId}`)
```